src/nxdoc/nuxeo-server/platform-services/preview.md

```diff
@@ -190,6 +190,29 @@ if (mtPreviewer != null) {
 }
 ```
 
+### ZipFile Handling Details
+
+When previewing a zip, the files it contains are listed. The displayed filenames are decoded as `UTF-8`. The following exception can be raised at `ZipFile`s instanciation if the underlying zip was encoded in a charset different from `UTF-8` containing non-ASCII characters:
+
+```java
+ZipException: invalid CEN header (bad entry name)
+```
+> Character set detection is at best an imprecise operation.
```

Comment:
Here I'm quoting Apache Tika, "at best" gives a ceiling to expectations.
I'd rather not change it if it's not grammatically incorrect.

Reply:
Hi, It's because I'm trying to work according to the Hyland documentation style guide as we are planning to unify all documentation. As it is grammatically correct, we can leave it for now. Thanks for the review.